packages/atomic/cypress/integration/search-box/instant-results/search-box-instant-results.cypress.ts

```diff
@@ -10,6 +10,9 @@ import * as SearchBoxAssertions from '../search-box-assertions';
 import * as InstantResultsAssertions from './search-box-instant-results-assertions';
 import {InstantResultsSelectors} from './search-box-instant-results-selectors';
 
+const delay = (force = false) => ({delay: 200, force});
```

Comment:
If the goal is to make sure some results are rendered every time, then I would avoid using a delay and I would instead assert that suggestions have changed after every character. A good example would be `typeFacetSearchQuery` in `facet-common-actions`.

Reply:
![Screenshot 2022-06-27 at 16 54 51](https://user-images.githubusercontent.com/30511433/175982544-dcf4a650-880c-4b0f-9de4-29f5e019d5dc.png)
ok, so I could try doing that for example, but the problem is that every time I call `.type`, it blurs the element and I lose the queries panel, so it doesn't work...
I think it will need to be delay for this one